package.json

```diff
@@ -11,10 +11,13 @@
   "scripts": {
     "api-gen": "./hack/gen-api-docs.sh",
     "build": "antora generate --clean --fetch antora-playbook.yml",
-    "preview:netlify": "netlify deploy --dir public"
+    "preview:netlify": "netlify deploy --dir public",
+    "lint:docs": "./hack/vale-check.sh",
+    "lint:docs:fix": "./hack/vale-check.sh",
```

Comment:
what's the difference between there two? Why we have fix?

Reply:
No difference. The fix target was added accidentally.